app/views/cms/homepage/_hero.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="home-hero">
```

Comment:
As mentioned, the style of this component is being overridden on my local version, so I would suggest to deploy this to staging and see how it looks like

Reply:
Removed manually from the CMS content, using the code editor view